x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/fix_logs_step/fix_logs_step.test.tsx

```diff
@@ -67,20 +63,36 @@ describe('Overview - Fix deprecation logs step', () => {
     });
 
     test(`It's incomplete when there are deprecation logs since last checkpoint`, async () => {
-      httpRequestsMockHelpers.setUpdateDeprecationLoggingResponse(getLoggingResponse(true));
+      httpRequestsMockHelpers.setLoadDeprecationLogsCountResponse({ count: 5 });
 
-      httpRequestsMockHelpers.setLoadDeprecationLogsCountResponse({
-        count: 5,
+      await act(async () => {
+        testBed = await setupOverviewPage();
       });
 
+      const { exists, component } = testBed;
+
+      component.update();
+
+      expect(exists(`fixLogsStep-incomplete`)).toBe(true);
+    });
+
+    test(`It's incomplete when log collection is disabled `, async () => {
```

Comment:
I smile every time I open a PR like this and the first thing I see are the new tests that we added! 😃